internal/controller/api/v1alpha1/nmstate_config_types.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+type RawState []byte
+
+type NMStateConfigSpec struct {
+	// One of the MAC addresses for the host that this network configuration is
```

Comment:
Suggest: `MACAddress is a MAC address for any network device on the host to which this config should be applied. This value is only used to ensure that the config is applied to the intended host.`